exercises/practice/scrabble-score/scrabble_score_test.rb

```diff
@@ -43,6 +43,6 @@ def test_scores_are_case_insensitive
 
   def test_convenient_scoring
     skip
-    assert_equal 13, Scrabble.score('alacrity')
+    assert_equal 13, Scrabble.new('alacrity').score
```

Comment:
Does the solution change because of this?

Reply:
I suppose so. The core logic required to solve doesn't change, and any solutions to the current form would still pass as the class method would now just be ignored. You just wouldn't need to have a separate both a class and instance method. It occurs to me now that might have been intentional to highlight the difference. I didn't think of it that way before.

Reply:
Yes, indeed, I think you have found the right path.